MetaMask/metamask-extension
# Add static files for the Flask build

Static files have been added for the Flask build. This includes logos of each size and variety that we use, and it includes the 3D model JSON file.

Manual testing steps:  
  - Create a Flask build and ensure the logo is correct (e.g. with the command `yarn build dev --build-type flask`)

Closes #12427
